Shipr-Helm/templates/redis-deployment.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: apps/v1
+kind: {{ .Values.redis.kind }}
+metadata:
+  name: {{ .Values.redis.name }}
+  labels:
+    app: {{ .Values.redis.name }}
+spec:
+  replicas: {{ .Values.redis.replicas }}
+  selector:
+    matchLabels:
+      app: {{ .Values.redis.name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.redis.name }}
+    spec:
+      containers:
+      - name: {{ .Values.redis.name }}
+        image: {{ .Values.redis.repository }}:{{ .Values.redis.tag }}
+        ports:
+        - containerPort: {{ .Values.redis.containerPort }}
```

Comment:
PodSpec is missing a lot of important fields, have a look here
https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec